framework/wazuh/core/common.py

```diff
@@ -213,11 +213,17 @@ def get_context_cache() -> dict:
 WAZUH_LIB = VAR_LIB / WAZUH_SERVER
 
 WAZUH_QUEUE = WAZUH_RUN / 'cluster'
-WAZUH_SOCKET = WAZUH_RUN / 'socket'
 
 WAZUH_SHARED = WAZUH_ETC / 'shared'
 
 LOCAL_SERVER_SOCKET = 'local-server.sock'
+LOCAL_SERVER_SOCKET_PATH = WAZUH_RUN / LOCAL_SERVER_SOCKET
+
+SERVER_UNIX_SOCKET = 'config-server.sock'
+SERVER_UNIX_SOCKET_PATH = WAZUH_RUN / SERVER_UNIX_SOCKET
```

Comment:
Maybe we could omit the `UNIX` part since it may be implicit, same for the comms API socket

```suggestion
CONFIG_SERVER_UNIX_SOCKET = 'config-server.sock'
CONFIG_SERVER_UNIX_SOCKET_PATH = WAZUH_RUN / CONFIG_SERVER_UNIX_SOCKET
```